src/app/cube/collections/collections.component.ts

```diff
@@ -0,0 +1,26 @@
+import { Component, OnInit } from '@angular/core';
+import { CollectionService } from 'app/core/collection.service';
+
+@Component({
+  selector: 'clark-collections',
+  templateUrl: './collections.component.html',
+  styleUrls: ['./collections.component.scss']
+})
+export class CollectionsComponent implements OnInit {
+
+  collections;
```

Comment:
Can we add Typescript Typings to this variable for consistency and readability?

Reply:
Updated to type collection, defined in the collection service.